### ISSUE TITLE
Failed to bind properties under 'spring.datasource.password' to java.lang.String

### ISSUE THREAD

**Issue by cobar79** [ulisesbocchio/jasypt-spring-boot#238]
Boot 2.3.2, jasypt 3.0.3 and 2.1.2 regardless of iv-generator-classname NoIvGenerator or RandomIvGenerator
`jasypt:
encryptor:
password: fubar
iv-generator-classname: org.jasypt.RandomIvGenerator

spring:
password: ENC(yaddayaddayadda)
`

With or without @EnableAutoConfiguration
@SpringBootApplication @EnableAutoConfiguration

Currently working in boot 2.2.4, jasypt 3.0.2 with encryptor.password in database

**Comment by miclefebvre** [comment]
I got the same error. 

It seems jasypt-spring-boot is not compatible with the new release of spring boot.

**Comment by startjava** [comment]
same to me!

**Comment by cobar79** [comment]
After painful research I changed the debug level on spring and found that the issue was jasypt couldn't decrypt the password that I had encrypted using the online tool.
Using jasypt 3.0.3 there is no command line CLI in the jar. The online tool doesn't work. 
Added:
``
            <plugin>
                <groupId>com.github.ulisesbocchio</groupId>
                <artifactId>jasypt-maven-plugin</artifactId>
                <version>3.0.3</version>
            </plugin>  
``
Run:
``
mvn jasypt:encrypt-value -Djasypt.encryptor.password="the password" -Djasypt.plugin.value="theValueYouWantToEncrypt"
``
